tests/test_services.py

```diff
@@ -0,0 +1,44 @@
+import os
+import time
+import json
+import unittest
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from utils import TestUtilities
+
+
+class TestServices(TestUtilities):
+
+    def setUp(self):
+        config_file = os.path.join(os.path.dirname(__file__), "config.json")
+        with open(config_file) as json_file:
+            self.config = json.load(json_file)
+        self.chrome_options = Options()
+        self.chrome_options.add_argument('--ignore-certificate-errors')
+        if self.config['headless']:
+            self.chrome_options.add_argument("--headless")
+        self.base_driver = webdriver.Chrome(options=self.chrome_options)
+
+    def cleanUp(self):
+        self.base_driver.close()
+
+    def test_services(self):
```

Comment:
This can be put before the admin login test. Why does this have a seperate class.

For that fact, I just realized, why are you creating seperate class for each test, all the tests can be in one class, if there is a reason you are seperating them in files, please let me know! :smile: